packages/foundations-vars/src/_colors.css

```diff
@@ -141,6 +141,7 @@
   --color-blue-light-8: #80b7ff;
   --color-blue-light-9: #cfe3ff;
   --color-blue-light-10: #eef3fc;
+  --color-blue-light-11: #92d8fe;
```

Comment:
Can you chat to @matt-tyas about this? We have a scale from 1–10 so 11 is a bit… 🤯